### PR TITLE
DEX-677 A little refactoring for CancelOrderTestSuite 

### PR DESCRIPTION
**Pull request by ngadiyak**

### CODE REVIEW

File: dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala [hunk 1]
```diff
@@ -202,33 +200,25 @@ class CancelOrderTestSuite extends MatcherSuiteBase {
...
-
       val amount             = 835.85722414.waves
-      val matcherFee         = 0.003.waves
       val traderTotalBalance = amount + matcherFee
-      broadcastAndAwait(mkTransfer(alice, trader, traderTotalBalance, Waves))
+      val trader             = createAccountWithBalance(traderTotalBalance -> Waves)
 
-      eventually {
-        dex1.api.tradableBalance(trader, wavesUsdPair).getOrElse(Waves, 0L) shouldBe traderTotalBalance
-      }
```
Comment (vsuharnikov) [inline dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala:215]: These lines guarantee that the account has Waves. Without them the test fails on my PC. You could move it to `createAccountWithBalance` and then remove such lines in CancelOrderTestSuite.

File: dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala [hunk 2]
```diff
@@ -202,33 +200,25 @@ class CancelOrderTestSuite extends MatcherSuiteBase {
...
-      val counterOrder   = mkOrder(trader, wavesUsdPair, OrderType.SELL, amount, 903200, matcherFee, version = 3)
-      val submittedOrder = mkOrder(alice, wavesUsdPair, OrderType.BUY, 0.0001.waves, 909700, matcherFee)
+      val counterOrder   = mkOrderDP(trader, wavesUsdPair, OrderType.SELL, amount, 9032, version = 3)
+      val submittedOrder = mkOrderDP(alice, wavesUsdPair, OrderType.BUY, 0.0001.waves, 9097)
 
       placeAndAwaitAtDex(counterOrder)
-      dex1.api.place(submittedOrder)
-
-      wavesNode1.api.waitForHeightArise()
+      placeAndAwaitAtNode(submittedOrder)
```
Comment (vsuharnikov) [inline dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala:214]: 👍

File: dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala (hunk 1 above)
Comment (ngadiyak) [reply dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala:215]: inside createAccountWithBalance we have broadcastAndAwait, it's enough for all other tests

Comment (ngadiyak) [reply dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala:215]: reverted this change

**Pull request merged**